### PR TITLE
[QPE-439] Values from scale is sometimes hard/impossible to see.

### PR DESCRIPTION
**Pull request by ahmed-Bazzara**
JIRA ticket link

https://tretton37.atlassian.net/secure/RapidBoard.jspa?rapidView=33&projectKey=QPE&modal=detail&selectedIssue=QPE-439

### CODE REVIEW

File: src/radarChart.js [hunk 1]
```diff
@@ -105,6 +105,12 @@ function displayRADAR(id, options, $element, layout, data, self) {
 
   //Wrapper for the grid & axes
   var axisGrid = g.append("g").attr("class", "axisWrapper");
+  var axisContainer = g.append('g').attr('class','axisContainer');
+  var blobWrapper = g.selectAll(".radarWrapper")
+    .data(data)
+    .enter().append("g")
+    .attr("class", "radarWrapper");
+  var axisLebelWrapper = g.append('g').attr('class', 'axsisLabelWrapper');
```
Comment (kristofferlind) [inline src/radarChart.js]:
```suggestion
  var axisLabelWrapper = g.append('g').attr('class', 'axsisLabelWrapper');
```

File: src/radarChart.js [hunk 2]
```diff
@@ -119,23 +125,23 @@ function displayRADAR(id, options, $element, layout, data, self) {
     .style("filter" , "url(#glow)");
 
   //Text indicating at what % each level is
-  axisGrid.selectAll(".axisLabel")
+  axisLebelWrapper.selectAll(".axisLabel")
```
Comment (kristofferlind) [inline src/radarChart.js]:
```suggestion
  axisLabelWrapper.selectAll(".axisLabel")
```

File: src/radarChart.js [hunk 3]
```diff
@@ -176,10 +182,7 @@ function displayRADAR(id, options, $element, layout, data, self) {
   }
 
   //Create a wrapper for the blobs
-  var blobWrapper = g.selectAll(".radarWrapper")
-    .data(data)
-    .enter().append("g")
-    .attr("class", "radarWrapper");
+
```
Comment (kristofferlind) [inline src/radarChart.js]:
```suggestion
```

File: src/radarChart.js [hunk 4]
```diff
@@ -176,10 +182,7 @@ function displayRADAR(id, options, $element, layout, data, self) {
   }
 
   //Create a wrapper for the blobs
```
Comment (kristofferlind) [inline src/radarChart.js]:
```suggestion
```

Comment (ahmed-Bazzara) [reply src/radarChart.js]: original author's comment

**Pull request merged**